compiler+runtime/test/jank/form/var/pass-symbol-on-var.jank

```diff
@@ -0,0 +1,3 @@
+(assert (= 'clojure.core/+ (symbol #'+)))
+
+:success
```

Comment:
These tests are for the compiler (lexer, parser, analysis, eval). The bug was in `clojure.core/symbol`, so the test should be in clojure.core-test.

Looks like we don't have a case there for vars. Up for adding it? https://github.com/jank-lang/clojure.core-test/blob/main/test/clojure/core_test/symbol.cljc

Reply:
Sure I can do that. IIUC this PR will then not include any test validating the fix.

Reply:
https://github.com/jank-lang/clojure.core-test/pull/66